include/votca/xtp/bse.h

```diff
@@ -144,9 +143,9 @@ class BSE {
   Interaction Analyze_eh_interaction(const QMStateType& type,
                                      const Orbitals& orb) const;
   template <typename BSE_OPERATOR>
-  Eigen::VectorXd Analyze_IndividualContribution(const QMStateType& type,
-                                                 const Orbitals& orb,
-                                                 const BSE_OPERATOR& H) const;
+  Eigen::VectorXd ExpectationValue_Operator(const QMStateType& type,
```

Comment:
strictly speaking the signature should be something like `const tools::EigenSystem`,const BSE_OPERATOR& H)`